backends-velox/src/main/scala/org/apache/spark/sql/execution/VeloxColumnarWriteFilesExec.scala

```diff
@@ -254,20 +255,28 @@ class VeloxColumnarWriteFilesRDD(
   }
 }
 
-class VeloxColumnarWriteFilesExec(
+case class VeloxColumnarWriteFilesExec(
```

Comment:
@zhztheplayer Could you also add a comment at the class level explaining why it is necessary to extend BinaryExecNode?